ui/src/components/import/ColumnDef.js

```diff
@@ -284,49 +343,73 @@ export const ExtendedSize = [
     {
         field: '450',
         editable: true,
+        width: 105,
+
     },
     {
         field: '500',
         editable: true,
+        width: 105,
+
     },
     {
         field: '550',
         editable: true,
+        width: 105,
+
     },
     {
-        field: '600',
+        field: '800',
```

Comment:
Was this intentional?   There's an 800 below

Reply:
fixed , mistake